src/types/task.ts

```diff
@@ -93,5 +93,5 @@ export enum TaskStatus {
   StatusBlocked = "blocked",
   Aborted = "aborted",
 
-  Known = "known-failure",
+  Known = "known-issue",
```

Comment:
nit. Can we change this name to `KnownIssue` i think it reads better as `TaskStatus.KnownIssue` as opposed to `TaskStatus.Known`

Reply:
done